libs/internals/core.write_compiled_resource.php

```diff
@@ -6,6 +6,46 @@
  */
 
 /**
+ * Check to see if the path is writable, works on Windows platforms
+ * Based from http://php.net/manual/en/function.is-writable.php comment by
+ * legolas558
+ *
+ * Added 04/03/2015 by Yola to address is_writable PHP bug that does not
+ * correctly handle ACL on Windows platforms.
```

Comment:
We use git. You don't need to write a changelog above the function. All we need to do is say that it's a `is_writeable` that works on Windows.